src/main/scala/scalachecklib/GeneratorsSection.scala

```diff
@@ -0,0 +1,220 @@
+  * combinators in the `Gen` object can be used to create or modify the behaviour of such generator functions.
+  *
+  * @param name generators
+  */
+object GeneratorsSection extends Checkers with Matchers with org.scalaexercises.definitions.Section {
+
+  import GeneratorsHelper._
+
+  /** Lets see how to create a new generator. The best way to do it is to use the generator combinators that exist
+    * in the `org.scalacheck.Gen` module. These can be combined using a for-comprehension. Suppose you need a generator
+    * which generates a tuple that contains two random integer values, one of them being at least twice as big as the
+    * other. The following definition does this:
+    */
+  def forComprehension(res0: Boolean) = {
+
+    import org.scalacheck.Gen
+    import org.scalacheck.Prop.forAll
+
+    val myGen = for {
+      n <- Gen.choose(10, 20)
+      m <- Gen.choose(2 * n, 500)
+    } yield (n,m)
+
+    check {
+      forAll(myGen) {
+        case (n, m) => (m >= 2 * n) == res0
+      }
+    }
+
+  }
+
+  /** You can create generators that picks one value out of a selection of values.
+    * The `oneOf` method creates a generator that randomly picks one of its parameters each time it generates a value.
+    * Notice that plain values are implicitly converted to generators (which always generates that value) if needed.
+    *
+    *
+    * The following generator generates a vowel:
+    */
+  def genOf(res0: Seq[Char]) = {
+
+    import org.scalacheck.Gen
+    import org.scalacheck.Prop.forAll
+
+    val vowel = Gen.oneOf('A', 'E', 'I', 'O', 'U')
+
+    val validChars: Seq[Char] = res0
+
+    check(forAll(vowel) { v =>
+      validChars.contains(v)
+    })
+  }
+
+  /** The distribution is uniform, but if you want to control it you can use the frequency combinator:
+    *
+    * {{{
+    * val vowel = Gen.frequency(
+    *   (3, 'A'),
+    *   (4, 'E'),
+    *   (2, 'I'),
+    *   (3, 'O'),
+    *   (1, 'U')
+    * )
+    * }}}
+    *
+    * Now, the vowel generator will generate ''E:s'' more often than ''U:s''. Roughly, 4/14 of the values generated
+    * will be ''E:s'', and 1/14 of them will be ''U:s''.
+    *
+    * Another methods in the `Gen` API:
+    * {{{
+    * def alphaChar: Gen[Char]
+    *
+    * def alphaStr: Gen[String]
+    *
+    * def posNum[T](implicit n: Numeric[T]): Gen[T]
+    *
+    * def listOf[T](g: Gen[T]): Gen[List[T]]
+    *
+    * def listOfN[T](n: Int, g: Gen[T]): Gen[List[T]]
+    * }}}
+    */
+  def genAPI(res0: Boolean, res1: Boolean, res2: Int) = {
+
+    import org.scalacheck.Gen.{alphaChar, posNum, listOfN}
+    import org.scalacheck.Prop.forAll
+
+    check(forAll(alphaChar)(_.isDigit == res0))
+
+    check(forAll(posNum[Int])(n => (n > 0) == res1))
+
+    check {
+      forAll(listOfN(10, posNum[Int])) { list =>
+        !list.exists(_ < 0) && list.length == res2
+      }
+    }
+  }
+
+  /** '''Conditional Generators'''
```

Comment:
I guess these single quotes `'` get rendered into headings, am i right? I believe we've used the equal sign `=` elsewhere for this.